virtio-devices/src/balloon.rs

```diff
@@ -94,6 +95,59 @@ pub struct VirtioBalloonConfig {
     actual: u32,
 }
 
+#[derive(Clone, Debug)]
+struct PartiallyBalloonedPage {
+    addr: u64,
+    bitmap: Vec<u64>,
+}
+
+impl PartiallyBalloonedPage {
+    fn new() -> Self {
+        let size = get_page_size();
+        let len = ((size >> VIRTIO_BALLOON_PFN_SHIFT) + 63) / 64;
+        Self {
+            addr: 0,
+            bitmap: vec![0; len as usize],
```

Comment:
As mentioned above, initializing padding bits as `1` would be better.

Reply:
OK.